### ISSUE TITLE
Tiny Threat missing with last verson of Details (WOTLKC)

### ISSUE THREAD

**Issue by Rdyx** [Tercioo/Details-Damage-Meter#439]
**Game and Details! version**
WOTLKC 3.4.1 / WD 10408 RELEASE 3.4.1

**Describe the bug**
TinyThreat plugin missing. Check in Details_TinyThreat folder and saw it was missing some files (especially *Details_TinyThreat-Wrath.toc*).

Copy/Pasting `Details_TinyThreat-Wrath.toc` from the previous version into the new one managed to fix it.

**Lua Error**
**Screenshots**
**Combat Log**
**Other Information**

**Comment by Tercioo** [comment]
Have you tried to close and reopen the game before copying back 
"Details_TinyThreat-Wrath.toc"?

------ Original Message ------
From: "Nicolas T" ***@***.***>
To: "Tercioo/Details-Damage-Meter" 
***@***.***>
Cc: "Subscribed" ***@***.***>
Sent: 18-Jan-23 6:56:25 PM
Subject: [Tercioo/Details-Damage-Meter] Tiny Threat missing with last 
verson of Details (WOTLKC) (Issue #439)

**Comment by Rdyx** [comment]
Yep, tiny threat was not detected as addon and also not in details after the last update, copying the file from an older version fixed it somehow

**Comment by sthkn** [comment]
Interesting; closing the game and reopening it fixed it for me. /reload wasn't enough.
Thanks Tercioo!
